SampleCode/Python/sample_identify_cross_page_tables.ipynb

```diff
@@ -4,17 +4,28 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Identify cross-page tables based on rules\n",
+    "# Identify cross-page tables and merge them into one table based on rules\n",
     "\n",
-    "This sample demonstrates how to use the output of Layout model and some business rules to identify cross-page tables. Once idenfied, it can be further processed to merge these tables and keep the semantics of a table.\n",
+    "This sample demonstrates how to use the output of Layout model and some business rules to identify markdown cross-page tables. Once idenfied, it will be processed to merge these markdown tables and keep the semantics of a table.\n",
     "\n",
```

Comment:
it's not identify markdown cross-page tables, it's identify cross-page tables and merge these tables in the markdown output.

Reply:
Fixed.